### DISCUSSION TITLE
Where do i download the ROMS?

### DISCUSSION THREAD

**Discussion by BananaJet** [skyline-emu/skyline#2301] [Q&A]
I have the emulator running perfectly but i need the actual games for it....

**Comment by JERD001** [comment]
The emulator doesn't support piracy. The way to get games is buying them on your preferred store and dump them from your switch.

**Comment by ghost** [comment]
😖

**Comment by ghost** [comment]
I don’t know if posts on discussions can be deleted, but this needs to be deleted NOW.

**Comment by WinnerWind** [comment]
Why get an emulator if you don't have any games? Piracy is wrong.
